migrations/m160118_081152_add_timetosend_column.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use yii\db\Migration;
```

Comment:
Missing `use nterms\mailqueue\MailQueue;` here to use `Yii::$app->get(MailQueue::NAME)->table`.